README.md

```diff
@@ -1,178 +1,176 @@
-(partitioning key), as well as full SQL support.
+<h3>TimescaleDB is an extension for PostgreSQL that enables time-series, events, and analytics workloads, while increasing ingest, query, and storage performance</h3>
 
-If you prefer not to install or administer your instance of TimescaleDB, try the
-30 day free trial of [Timescale Cloud](https://console.cloud.timescale.com/signup), our fully managed cloud offering.
-Timescale is pay-as-you-go. We don't charge for storage you dont use, backups, snapshots, ingress or egress.
+[![Docs](https://img.shields.io/badge/Read_the_Timescale_docs-black?style=for-the-badge&logo=readthedocs&logoColor=white)](https://docs.timescale.com/)
+[![SLACK](https://img.shields.io/badge/Ask_the_Timescale_community-black?style=for-the-badge&logo=slack&logoColor=white)](https://timescaledb.slack.com/archives/C4GT3N90X)
+[![Try TimescaleDB for free](https://img.shields.io/badge/Try_Timescale_for_free-black?style=for-the-badge&logo=timescale&logoColor=white)](https://console.cloud.timescale.com/signup)
 
-To determine which option is best for you, see [Timescale Products](https://tsdb.co/GitHubTimescaleProducts)
-for more information about our Apache-2 version, TimescaleDB Community (self-hosted), and Timescale
-Cloud (hosted), including: feature comparisons, FAQ, documentation, and support.
+</div>
 
-Below is an introduction to TimescaleDB. For more information, please check out
-these other resources:
-- [Developer Documentation](https://docs.timescale.com/getting-started/latest/services/)
-- [Slack Channel](https://slack-login.timescale.com)
-- [Timescale Community Forum](https://www.timescale.com/forum/)
-- [Timescale Release Notes & Future Plans](https://tsdb.co/GitHubTimescaleDocsReleaseNotes)
+TimescaleDB scales PostgreSQL for time-series data with the help of [hypertables](https://docs.timescale.com/use-timescale/latest/hypertables/about-hypertables/). Hypertables are PostgreSQL tables that automatically partition your data by time and space. You interact with a hypertable in the same way as regular PostgreSQL table. Behind the scenes, the database performs the work of setting up and maintaining the hypertable's partitions.
 
-For reference and clarity, all code files in this repository reference
-licensing in their header (either the Apache-2-open-source license
-or [Timescale License (TSL)](https://github.com/timescale/timescaledb/blob/main/tsl/LICENSE-TIMESCALE)
-). Apache-2 licensed binaries can be built by passing `-DAPACHE_ONLY=1` to `bootstrap`.
+From the perspective of both use and management, TimescaleDB looks and feels like PostgreSQL, and can be managed and queried as
+such. However, it provides a range of features and optimizations that make managing your time-series data easier and more efficient.
 
-[Contributors welcome.](https://github.com/timescale/timescaledb/blob/main/CONTRIBUTING.md)
+| **Get started with TimescaleDB** |**Learn more about TimescaleDB**|
```

Comment:
aesthetics nit: can we turn this into two lists without a table putting them side-by-side? I appreciate the effort to wrap it, tables in MD are hard to wrestle and vertical alignment is off. Having the lists below each other with the headers is also better for SEO.

Reply:
Hi @philkra , the idea behind this table is that when people read the page they see the things they can do, and the first major action (Install), https://github.com/timescale/timescaledb/blob/bfe181d5444e9c4a1cf3fd3ce5ea54beccfcc01f/README.md (Yes, we need to sort out the links).  If you put one list after another then you see two lists.  Your call.